### ISSUE TITLE
JDK 9 support

### ISSUE THREAD

**Issue by andrcuns** [prashant-ramcharan/courgette-jvm#46]
Given that java 10 is already out, I think it is worth to think about at least java 9 support. Currently execution will fail with error:
```
Caused by: java.lang.ClassCastException: java.base/jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to java.base/java.net.URLClassLoader
        at courgette.runtime.CourgetteFeatureRunner.<clinit>(CourgetteFeatureRunner.java:56)
```
https://blog.codefx.org/java/java-9-migration-guide/#Casting-To-URL-Class-Loader

**Comment by prashant-ramcharan** [comment]
Yes, will into supporting Java 1.9+

**Comment by prashant-ramcharan** [comment]
@andrcuns This is now added to 2.3.0-beta which supports JDK10 (_and JDK9_)

Please try and let me know if any issues.

````gradle
compile 'io.github.prashant-ramcharan:courgette-jvm:2.3.0-beta'
````

[Changelog](https://bintray.com/prashantr/Courgette-JVM/courgette-jvm/2.3.0-beta#release)

**Comment by prashant-ramcharan** [comment]
This is now released in 2.3.1
